examples/se2_localization.cpp

```diff
@@ -280,9 +280,11 @@ int main()
         //// IV. Results ##############################################################################
 
         // DEBUG
-        cout << "X simulated : " << X_simulation.log().coeffs().transpose() << endl;
-        cout << "X estimated : " << X.log().coeffs().transpose() << endl;
-        cout << "X unfilterd : " << X_unfiltered.log().coeffs().transpose() << endl;
+        cout << "X simulated : " << X_simulation.log().coeffs().transpose() << "." << endl;
+        cout << "X estimated : " << X.log().coeffs().transpose()
+             << ". |d|=" << (X_simulation - X).squaredWeightedNorm() <<endl;
+        cout << "X unfilterd : " << X_unfiltered.log().coeffs().transpose()
+             << ". |d|=" << (X_simulation - X_unfiltered).squaredWeightedNorm() <<endl;
```

Comment:
why squared weighted norm? If you name it `|d|` should be the norm, not squared norm